modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -153,7 +153,9 @@ export default {
       onUpdate: this.editorUpdate,
       extensions: [
         StarterKit,
-        TextAlign,
+        TextAlign.configure({
+          types: [ 'heading', 'paragraph' ]
```

Comment:
Probably okay but we already keep a list of all the tiptap types apostrophe cares about, consider using this to keep our configuration in one place
```
TextAlign.configure({
  types: Object.keys(this.tiptapTypes)
}),
```